### DISCUSSION TITLE
Preview images for blog posts? (And other typical blog functions)

### DISCUSSION THREAD

**Discussion by drwilkins** [thuliteio/doks#500] [Ideas]
Is there a way to add a preview image to the blog posts listed on an index page?
e.g. here:
![image](https://user-images.githubusercontent.com/6098768/136673076-91c766e7-2e0f-4789-9228-6b6a180b446e.png)

In general I *love* doks--thank you! I especially like the way it combines documentation and blog functions. I'm just looking for a few other options for the blog aspect--e.g. image thumbnails, tag list, and recommendations based on tags. Any of these additional features would be really valuable.

**Comment by h-enk** [comment]
Thanks!

> I'm just looking for a few other options for the blog aspect--e.g. image thumbnails, tag list, and recommendations based on tags. Any of these additional features would be really valuable.

Recorded (for the next version): https://github.com/h-enk/doks/issues/501

**Comment by drwilkins** [comment]
Awesome, thanks, Henk!

**Comment by engeir** [comment]
What is the syntax for adding preview images to a blog post?

Having
```yaml
---
...
images: ['https://via.placeholder.com/300x150']
...
---
```
(or any other image url) just give me the default white rectangle.

**Reply by engeir** [reply]
Ok I figured it out: image should be next to the index.md file in the directory for the blog post.
